package.json

```diff
@@ -6,7 +6,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint:js": "next lint",
```

Comment:
I had to make sure I had all of these installed too! Once I ran `npm install`, I got the correct error message!